control/Controller.php

```diff
@@ -127,10 +127,11 @@ public function doInit() {
 	/**
 	 * Returns a link to this controller. Overload with your own Link rules if they exist.
 	 *
+	 * @param string $action Optional action
 	 * @return string
 	 */
-	public function Link() {
-		return get_class($this) .'/';
+	public function Link($action = null) {
+		return Controller::join_links(get_class($this), $action, '/');
```

Comment:
1) We don't want all args, just the one. See tests below; I'm relying on $action being ignored if empty.
2) Modifying join_links could have a larger range of unexpected consequences, so I've opted for the least disruptive change.